unstract/core/src/unstract/core/utilities.py

```diff
@@ -23,3 +23,9 @@ def get_env(env_key: str, default: Optional[str] = None, raise_err=False) -> str
         if env_value is None or env_value == "" and raise_err:
             raise RuntimeError(f"Env variable {env_key} is required")
         return env_value
+
+    @staticmethod
+    def build_tool_container_name(
+        tool_image: str, tool_version: str, run_id: str
+    ) -> str:
+        return f"{tool_image.split('/')[-1]}-{tool_version}-{run_id}"
```

Comment:
We need to make sure the total length doesn't exceed 63 chars. This is required for k8s

Reply:
I can add a check for this in a separate PR but this will be guaranteed for all our existing tool image names now unless we introduce an excessively long name

Reply:
Raised https://github.com/Zipstack/unstract/pull/697